README.md

```diff
@@ -101,3 +101,20 @@ We use [Google Java format](https://github.com/google/google-java-format) to for
 ```bash
 ./format.sh
 ````
+
+# Snowpipe Streaming Example (Still in Preview)
```

Comment:
Could we link https://docs.snowflake.com/en/LIMITEDACCESS/snowpipe-streaming.html somewhere?